.coveragerc

```diff
@@ -6,6 +6,7 @@ omit =
     */MDAnalysis/tests/*
     */legacy/*
     */due.py
+concurrency = thread, multiprocessing
```

Comment:
Since we use some multiprocessing in our code, we should include this I think (see: https://coverage.readthedocs.io/en/latest/config.html#run-concurrency)

Reply:
thanks for linking the docs, I agree